azure-functions-maven-plugin/src/test/java/com/microsoft/azure/maven/function/invoker/storage/EventHubProcesser.java

```diff
@@ -0,0 +1,152 @@
+
+        final boolean isEventHubNamespaceExist = azureClint.eventHubNamespaces().list().stream()
+                .anyMatch(namespace -> namespace.name().equals(namespaceName) &&
+                        namespace.resourceGroupName().equals(resourceGroupName));
+        if (isEventHubNamespaceExist) {
+            eventHubNamespace = azureClint.eventHubNamespaces().getByResourceGroup(resourceGroupName, namespaceName);
+        } else {
+            eventHubNamespace = azureClint.eventHubNamespaces()
+                    .define(namespaceName).withRegion(resourceGroup.region())
+                    .withExistingResourceGroup(resourceGroupName).create();
+        }
+
+        storageAccount = azureClint.storageAccounts().getByResourceGroup(resourceGroupName, storageAccountName);
+        if (storageAccount == null) {
+            storageAccount = azureClint.storageAccounts()
+                    .define(storageAccountName).withRegion(resourceGroup.region())
+                    .withExistingResourceGroup(resourceGroup)
+                    .withSku(StorageAccountSkuType.STANDARD_LRS)
+                    .withGeneralPurposeAccountKindV2().create();
+        }
+    }
+
+    public EventHub createOrGetEventHubByName(String eventHubName) throws Exception {
+        final Azure azureClient = CommonUtils.getAzureClient();
+        final List<EventHub> resultList = eventHubNamespace.listEventHubs().stream()
+                .filter(eventHub -> eventHub.name().equals(eventHubName)).collect(Collectors.toList());
+        if (resultList.size() == 0) {
+            return azureClient.eventHubs().define(eventHubName).withExistingNamespace(eventHubNamespace)
+                    .withExistingStorageAccountForCapturedData(storageAccount, eventHubName)
+                    .withDataCaptureEnabled().create();
+        } else {
+            return resultList.get(0);
+        }
+    }
+
+    public void sendMessageToEventHub(String eventhub, String message) throws Exception {
+        final EventHubClient ehClient = getEventHubClientByName(eventhub);
+        final String payload = message;
+        final Gson gson = new Gson();
+        final byte[] payloadBytes = gson.toJson(payload).getBytes(Charset.defaultCharset());
+        final EventData sendEvent = EventData.create(payloadBytes);
+        ehClient.send(sendEvent).get();
+    }
+
+    public List<String> getMessageFromEventHub(String eventhub) throws Exception {
+        final List<String> result = new ArrayList<>();
+        final EventHubClient ehClient = getEventHubClientByName(eventhub);
+        final List<String> partitionIds = Arrays.asList(ehClient.getRuntimeInformation().get().getPartitionIds());
+        partitionIds.parallelStream()
+                .forEach(partitionId -> result.addAll(receiveMessageFromSpecificPartition(ehClient, partitionId)));
+        return result;
+    }
+
+    public List<String> receiveMessageFromSpecificPartition(EventHubClient eventHubClient, String partitionId) {
```

Comment:
Add `@NotNull` to the parameters to leverage the IDE NPE check. And better to add the `final` modifier, even though these are test code, it is better if we apply the same style with production code.